docs/docs/examples/guides/04-object-detection-example.md

```diff
@@ -0,0 +1,745 @@
+```
+
+Next, configure AICA Studio and add **objectdetection** under Custom Packages. After launching, you should see
+**Component Utils** under **Add Component** and be able to add *YOLO to Marker*.
+
+### Application Setup
+
+Connect the `Bounding Boxes` output of *YOLO Executor* to `JSON Input` on *YOLO to Marker* then configure it with
+**auto-configure** and **auto-activate**. You should also set the parameters based on your camera and what you are filming.
+Set `rate` to match that of *YOLO Executor*. Set the `Object to Track` parameter to "person" to track yourself in the frame. With the application running the logs should show the 3D position in the camera_frame.
+
+#### Creating a Frame and Converting it to a Signal
+
+Add in a *Hardware Interface* if it is not already there, and run the application. When it is running, select _3D Viz_ at
+the top right. Record a frame `tool0` and name it `camera_frame`, this is the position of the camera. It can be moved by
+dragging the axis markers, with the Z direction as the direction in which the camera is pointing. Note that the robot
+will move to where the objects are detected, which may be unreachable depending on the camera position, and cause the
+robot controller to stop.
+
+Stop the application, and go back to _graph_, open the yaml code and check that the recorded frame is there, for
+example:
+
+```yaml
+frames:
+  camera_frame:
+    reference_frame: world
+    position:
+      x: 0
+      y: 0.3
+      z: 0.4
+    orientation:
+      w: 0
+      x: 0
+      y: 1
+      z: 0
+```
+
+To convert the frame to a signal add *TF to Signal* from **AICA Core Components**, connect the **load** node to the *on start* block (purple square with a black play symbol), and configure the component to **auto-configure**, **auto-activate**, set `camera_frame` and `Reference frame` to world. With the application running the camera frame can be observed under
+`ROS Topics/tf_to_signal/pose(modulo_interfaces/msg/EncodedState)`.
+
+#### Tranforming a Frame to World
+
+To drive the robot towards the object position it needs to be in `world` frame. The *Cartesian Transformation* component can convert from `camera_frame` to `world`. Add it to the application and connect the load node. Finally, connect the `pose` output from *TF to Signal* to `Input 1`, and the output of *YOLO to Marker* to `Input 2`. The output of *Cartesian Transformation* is now the object position in the world frame.
+
+#### Moving a Robot Towards a Frame
+
+We can use a *Signal Point Attractor* to move the robot end effector towards the object. Add the component to the
+application, and set it to **auto-configure** and **auto-activate**. The load node can be connected to the `On Activate`
+transition in *YOLO to Marker*. This means that the robot will not move until after *YOLO to Marker* has started. The
+`Cartesian State`, under *Robot State Broadcaster* should be connected to the `Input State`_` of *Signal Point Attractor*,
```

Comment:
```suggestion
`Cartesian State`, under *Robot State Broadcaster* should be connected to the `Input State` of *Signal Point Attractor*,
```

not sure what the intention here is with text styling. In general and as mentioned before you'd need to reconsider the use of code blocks/bold/italics etc.